test/Microsoft.AspNetCore.SignalR.Microbenchmarks/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using BenchmarkDotNet.Running;
+
+namespace Microsoft.AspNetCore.SignalR.Microbenchmarks
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            if (args.Contains("--debug"))
```

Comment:
It is likely to cause some confusion in this case, since the Benchmarks themselves are run in separate processes.